apps/app/src/main/scala/org/lfdecentralizedtrust/splice/console/ConsoleHttpCommandRunner.scala

```diff
@@ -56,9 +53,10 @@ class ConsoleHttpCommandRunner(
         traceContext
       )
       val commandTimeout = commandTimeouts.bounded
-
+      // TODO check this.
+      val metricsFactory = environment.metricsRegistry.generateMetricsFactory(MetricsContext.Empty)
```

Comment:
I can probably get this from somewhere else (and all cases where I'm currently generating it from empty)